DCC.combined-schema.json

```diff
@@ -312,6 +329,12 @@
       "type": "string",
       "valueset-uri": "valuesets/vaccine-mah-manf.json"
     },
+    "country_vt": {
+      "description": "Country of Vaccination / Test, ISO 3166 alpha-2 where possible",
+      "type": "string",
+      "pattern": "[A-Z]{1,10}",
+      "valueset-uri": "valuesets/country-2-codes.json"
```

Comment:
How can the value-set be the country-2-codes, but then allow the specified pattern? Is the valueset just a recommendation? If not, how can one know if the valueset is mandatory or advisory?

Reply:
...because it isn't is the short answer

Filename comes from it being primarily ISO8601 alpha-2, but the formally approved eHN specification makes provision for it. So the valueset is a (small) superset of ISO8601 alpha-2
v1.3.0 spec:
> Country expressed as a 2-letter ISO3166 code (RECOMMENDED) or a reference to an international organisation responsible for the vaccination event (such as UNHCR or WHO). A coded value from the value set country-2-codes.json.